integreat_compass/cms/views/components/report_list_view.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...models import OfferVersion, Report
+
+
+@method_decorator(permission_required("cms.add_vote"), name="dispatch")
+class ReportListView(TemplateView):
+    """
+    View for the report list
+    """
+
+    template_name = "components/report_list.html"
+
+    def get_context_data(self, **kwargs):
+        r"""
+        Context for the report list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+        return super().get_context_data(**kwargs) | {"reports": Report.objects.all()}
```

Comment:
Same goes here: This query needs to be a bit different. I should be all offers who's latest version has been reported and not been rejected yet